deploy/buildtools/bitbuilder.py

```diff
@@ -695,7 +695,11 @@ def on_build_failure():
 
         # store bitfile (and mcs if it exists)
         local(f"cp {bit_path} {tar_staging_path}")
-        local(f"cp {mcs_path} {tar_staging_path}")
+        if self.build_config.PLATFORM != "xilinx_vcu118":
+            if self.BOARD_NAME == "mellanox_innova_2":
+                local(f"cp {local_cl_dir}/vivado_proj/firesim*.mcs {tar_staging_path}")
```

Comment:
Why is this needed? Shouldn't the MCS file be called `firesim.mcs`?

Reply:
This is needed because the mellanox uses two QSPI chips in parallel so you end up with a `firesim_primary.mcs` and a `firesim_secondary.mcs`. The VCU118 looks like it has a similar situation, but that code still expects that the first MCS file is `firesim.mcs`, hence the bit of hack